src/foam/nanos/cron/Cron.js

```diff
@@ -28,6 +28,11 @@ foam.CLASS({
   searchColumns: ['id', 'description'],
 
   properties: [
+    {
+      class: 'FObjectProperty',
+      of: 'foam.nanos.cron.Schedule',
+      name: 'schedule'
```

Comment:
Could you set the default value to a NeverSchedule singleton, then move the null pointer check below?

Reply:
The NeverSchedule singleton should disable a job, right? Right now the default behaviour (when schedule_==null) is to use the old Cron properties - this is my solution for making old DAO entries compatible.

Now that you have me thinking about it, I should be adapting the old properties into an instance of CronSchedule instead of keeping the now-duplicate logic in Cron.js.